src/init/starship.nu

```diff
@@ -1,27 +1,35 @@
-let-env STARSHIP_SHELL = "nu"
-let-env STARSHIP_SESSION_KEY = (random chars -l 16)
-let-env PROMPT_MULTILINE_INDICATOR = (^::STARSHIP:: prompt --continuation)
+export-env { load-env {
+    STARSHIP_SHELL: "nu"
+    STARSHIP_SESSION_KEY: (random chars -l 16)
+    PROMPT_MULTILINE_INDICATOR: (
+        ^::STARSHIP:: prompt --continuation
+    )
 
-# Does not play well with default character module.
-# TODO: Also Use starship vi mode indicators?
-let-env PROMPT_INDICATOR = ""
+    # Does not play well with default character module.
+    # TODO: Also Use starship vi mode indicators?
+    PROMPT_INDICATOR: ""
 
-let-env PROMPT_COMMAND = { ||
-    # jobs are not supported
-    let width = (term size).columns
-    ^::STARSHIP:: prompt $"--cmd-duration=($env.CMD_DURATION_MS)" $"--status=($env.LAST_EXIT_CODE)" $"--terminal-width=($width)"
-}
+    PROMPT_COMMAND: {||
+        # jobs are not supported
+        (
+            ^::STARSHIP:: prompt
+                --cmd-duration $env.CMD_DURATION_MS
+                $"--status=($env.LAST_EXIT_CODE)"
+                --terminal-width (term size).columns
```

Comment:
The `--flag=` syntax has the advantage that it may handle empty values, better I would prefer to keep using it.

Reply:
i think Nushell handles *empty values* out-of-the-box.
there is the `null` value which is passed if an argument is not defined, e.g. with the following mockup of the init script,
```bash
def starship [-a: any, -b: any, -c: any] {
    print $"a: ($a)"
    print $"b: ($b)"
    print $"c: ($c)"
}

def init [-a: any, -b: any, -c: any] {
    starship -a $a -b $b -c $c
}
```
`init ...` with any combination of options works just fine :relieved: 

also i think `$env.CMD_DURATION_MS`, `$env.LAST_EXIT_CODE` and `term size` will always return something :+1: 

i think this is safe and more clean, you tell me of course :yum: